src/Pages/Authentication/Login/Login.jsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react';
+import { Link } from "react-router-dom";
+import '../Css/Authentication.css';
+import Input from '../../../Shared/Input/Input';
+import blitheCollage from '../../../Assets/Images/Blithe-collage.jpg';
+import blitheLogo from '../../../Assets/Images/Blithe-logo.png';
+import InputField from '../../../Shared/Input/InputField';
+
+const Login = () =>{
+
+    const [showPassword , setShowPassword ] = useState(false);
+    const [loginData, setLoginData] = useState(
+        {   
+            email:'',
+            password:''  
+        })
+
+    //Handle show/hide password
+    const handlePassword = () =>{
+        setShowPassword(!showPassword)
+    }
+
+    const handleLogin = (e) =>{
+        e.preventDefault();
+        //commented right now for future perspective..
+        // console.log('LoginData', loginData);
+    }
+
+    return(
+        <>
+            <section className="auth">
+            <div className="col-7 auth-img-container">
+                    <img src={blitheCollage}
+                        alt="auth-img" />
+                </div> 
+                <div className="col-7 auth-content-container">
+                    <div className="auth-form">
+                        <img src={blitheLogo} className="blithe-logo" alt="blithe-logo"/>
+                        <form>
+                            <p className="heading-md heading">Login</p>
+                            <InputField 
+                                type='email'
+                                title='Email'
+                                name='email'
+                                placeholder='Enter email'
+                                value={loginData?.email? loginData.email : ''}
+                                onChange={(e)=>setLoginData(prevData => {return  {...prevData, email:e.target.value}})}
+                            />
+                             <InputField 
+                                type={showPassword?'text':'password'}
+                                title='Password'
+                                name='password'
+                                placeholder='Enter password'
+                                showPassword={showPassword}
+                                handlePassword = {handlePassword}
+                                value={loginData?.password? loginData.password : ''}
+                                onChange={(e)=>setLoginData(prevData => {return  {...prevData, password:e.target.value}})}
+                            />
+
+                            <div className="remember">
+                                <label>
+                                    <Input id='rememberUser' type="checkbox" name ='remember' checked={true}/>
+                                    Remember me!
+                                </label>
```

Comment:
Suggestion: use `htmlFor` attribute in label tag so clicking on text would checked checkbox

Reply:
ok